sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/implementation/models/ListQueuesSegmentResponse.java

```diff
@@ -188,4 +175,94 @@ public ListQueuesSegmentResponse setNextMarker(String nextMarker) {
         this.nextMarker = nextMarker;
         return this;
     }
+
+    @Override
+    public XmlWriter toXml(XmlWriter xmlWriter) throws XMLStreamException {
+        return toXml(xmlWriter, null);
+    }
+
+    @Override
+    public XmlWriter toXml(XmlWriter xmlWriter, String rootElementName) throws XMLStreamException {
+        rootElementName = CoreUtils.isNullOrEmpty(rootElementName) ? "EnumerationResults" : rootElementName;
+        xmlWriter.writeStartElement(rootElementName);
+        xmlWriter.writeStringAttribute("ServiceEndpoint", this.serviceEndpoint);
+        xmlWriter.writeStringElement("Prefix", this.prefix);
+        xmlWriter.writeStringElement("Marker", this.marker);
+        xmlWriter.writeIntElement("MaxResults", this.maxResults);
+        if (this.queueItems != null) {
+            xmlWriter.writeStartElement("Queues");
+            for (QueueItem element : this.queueItems) {
+                xmlWriter.writeXml(element, "Queue");
+            }
+            xmlWriter.writeEndElement();
+        }
+        xmlWriter.writeStringElement("NextMarker", this.nextMarker);
+        return xmlWriter.writeEndElement();
+    }
+
+    /**
+     * Reads an instance of ListQueuesSegmentResponse from the XmlReader.
+     *
+     * @param xmlReader The XmlReader being read.
+     * @return An instance of ListQueuesSegmentResponse if the XmlReader was pointing to an instance of it, or null if
+     *     it was pointing to XML null.
+     * @throws IllegalStateException If the deserialized XML object was missing any required properties.
+     * @throws XMLStreamException If an error occurs while reading the ListQueuesSegmentResponse.
+     */
+    public static ListQueuesSegmentResponse fromXml(XmlReader xmlReader) throws XMLStreamException {
```

Comment:
Are these `fromXml` and `toXml` methods generated?

Reply:
Yeah, these are generated